MvvmCross/MvvmCross.csproj

```diff
@@ -12,6 +12,7 @@
 This package contains the 'Core' libraries for MvvmCross</Description>
     <PackageId>MvvmCross</PackageId>
     <GenerateLibraryLayout Condition=" $(TargetFramework.StartsWith('net6.0-windows')) ">true</GenerateLibraryLayout>
+    <Nullable>enable</Nullable>
```

Comment:
Can you please revert this and instead opt into nullable with `#nullable enable` in the files where you want to use it.

If you enable nullable attributes project wise, it needs to be handled everywhere. There is still a lot of surface where it hasn't been adressed.

Reply:
Yeah, sure. I just couldn't wait to enable nullable everywhere :)

Reply:
Done